core/src/main/scala/kafka/server/DynamicBrokerConfig.scala

```diff
@@ -48,13 +48,6 @@ import scala.collection._
 import scala.jdk.CollectionConverters._
 
 /**
-  * Dynamic broker configurations are stored in ZooKeeper and may be defined at two levels:
```

Comment:
in kraft mode, the two levels are still existent. could you please revise it instead of removing it?

Reply:
I have a follow-up ask, while the two levels are still present they will no longer be under the path specified in this file since the path is ZK-specific. As long as we don't use this comment to generate documentation I am okay with this being removed in a subsequent PR. However, if we use this file to generate documentation could you change this as part of this PR?

Reply:
I think those comments in DynamicBrokerConfig are not used to generate documentation, but it will be better to correct it - as least, the description about zk path must be removed.

Reply:
Okay, then I will review once they have been changed 😊!

Reply:
Thanks for @clolov and @chia7712, update this document. If there is any idea, please let me know